packages/react-server/core/ClientController.js

```diff
@@ -55,6 +55,7 @@ class ClientController extends EventEmitter {
 
 		window.__reactServerTimingStart = window.performance ? window.performance.timing.navigationStart : new Date;
 		var wakeTime = new Date - window.__reactServerTimingStart;
+		var aboveTheFold = window.__displayAboveTheFold - window.__reactServerTimingStart;
```

Comment:
yeah just as wakeTime falls back to a zero-ish value, I'll just handle the edge case as 0. Seems more sensible than some large negative number and easier to filter out.